distribution/lib/Standard/Base/0.0.0-dev/src/Data.enso

```diff
@@ -51,6 +52,16 @@ from project.System.File_Format import Auto_Detect, File_Format
      If set to `Report_Error`, the operation fails with a dataflow error.
      If set to `Ignore`, the operation proceeds without errors or warnings.
 
+   ! Request Caching
+
+     Responses to HTTP data requests are cached, and additional requests for the
+     same resources will use the cache, saving a round-trip call to the remote
+     server. Two resources are considered the same if the URIs and request
+     headers are the same. Header order does not affect sameness.
+
+     The cached values are retained as long as the project remains open. Closing
+     a project will clear the cache.
```

Comment:
This ignores the TTL. Maybe we should mention that we rely on the `Max-Age` header, if it is present, to decide how long to cache the result?

Reply:
(like we do in `fetch`)

I know this was supposed to be the shorter doc, but we could at least have a short mention + pointer to read `Data.fetch` doc for more info.